src/client.js

```diff
@@ -12,14 +13,14 @@ const fragmentMatcher = new IntrospectionFragmentMatcher({
 
 const client = new ApolloClient({
   link: new HttpLink({
-    uri: 'https://platform-api-beta.opentargets.io/api/v4/graphql',
+    uri: config.urlAppolloClient,
```

Comment:
Would suggest to change this to a more generic name like `apiUrl` or something like that since Apollo is just a library among many to fetch data from a  GraphQL API.

Reply:
I have just refactored the input parameters' names as requested in this comment.